src/DurableTask.Netherite/TransportLayer/EventHubs/EventHubsConnections.cs

```diff
@@ -63,6 +68,11 @@ public async Task StartAsync(TaskhubParameters parameters)
                 this.EnsurePartitionsAsync(parameters.PartitionCount), 
                 this.EnsureClientsAsync(), 
                 this.EnsureLoadMonitorAsync());
+
+            if (this.consumerGroup != defaultConsumerGroup)
+            {
+                await this.EnsureConsumerGroupsExistAsync();
```

Comment:
Should we consider doing the Consumer Group Creation in the OrchestrationServiceCreateAsync call? since its a resource being created?

Reply:
I think that would not work beause the event hub itself is dynamically created (and sometimes deleted and recreated for certain recovery scenarios).